src/elements/logo/logo.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import Icon from '../icon'
+import styles from './logo.css'
+
+import { classNames } from 'utils'
+
+const Logo = React.memo(
+  ({
+    children = 'Dashboard',
+    className,
+    src = '#core-symbol',
+    alt = 'CORE',
```

Comment:
I think it's better to have it editable as it is but I am not sure. What's your opinion?

Reply:
It allows to use different logos, on the other hand, we apply custom negative margin on icon so any other icon can look badly 🤷‍♂️

Reply:
Let's keep it 🙂